src/main/java/com/englishtown/vertx/elasticsearch/SearchOptions.java

```diff
@@ -35,6 +35,9 @@
     private JsonObject aggregations;
     private List<SortOption> sorts = new ArrayList<>();
     private JsonObject extraSource;
+    private String templateName;
+    private String templateType;
```

Comment:
Any reason not to use the enum ScriptService.ScriptType rather than a String?
The code generator supports any Enum class (https://github.com/vert-x3/vertx-codegen).